Multisig/Logic/Models/Safe.swift

```diff
@@ -75,6 +75,19 @@ extension Safe: Identifiable {
 
         CoreDataStack.shared.saveContext()
     }
+    
+    static func delete(safe: Safe) {
```

Comment:
Ok, now, these new business logic methods do not have unit tests. Please add unit tests for them.